### PR TITLE
APP-792 Handle multiple stream containers

### PR DESCRIPTION
**Pull request by DTCurrie**
This updates our stream handling logic to handle scenarios where remote bases are added.

### CODE REVIEW

File: web/frontend/src/api.ts [hunk 1]
```diff
@@ -107,48 +106,42 @@ const connect = async (authEntity = savedAuthEntity, creds = savedCreds) => {
...
-        mediaElementPreview.controls = false;
-      } else {
-        mediaElementPreview.controls = true;
-      }
-
-      const streamPreviewContainer = document.querySelector(`#stream-preview-${streamName}`);
-      if (streamPreviewContainer) {
-        const child = streamPreviewContainer.querySelector(kind);
+      const streamPreviewContainers = document.querySelectorAll(`[data-stream-preview="${streamName}"]`);
+      console.log('streamPreviewContainers', streamPreviewContainers);
```
Comment (DTCurrie) [inline web/frontend/src/api.ts:131]:
```suggestion
```

File: web/frontend/src/lib/stream.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import streamApi from '../gen/proto/stream/v1/stream_pb.esm';
+
+export const addStream = async (name: string) => {
+  console.log('add stream', { name });
+  await new Promise<void>((resolve, reject) => {
+    const req = new streamApi.AddStreamRequest();
+    req.setName(name);
+    window.streamService.addStream(req, new grpc.Metadata(), (err) => {
+      if (err) {
+        console.log('add stream', { err });
```
Comment (DTCurrie) [inline web/frontend/src/lib/stream.ts:12]:
```suggestion
```

File: web/frontend/src/lib/stream.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        return;
+      }
+
+      resolve();
+    });
+  });
+};
+
+export const removeStream = async (name: string) => {
+  console.log('remove stream', { name });
```
Comment (DTCurrie) [inline web/frontend/src/lib/stream.ts:23]:
```suggestion
```

File: web/frontend/src/lib/stream.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  await new Promise<void>((resolve, reject) => {
+    const req = new streamApi.RemoveStreamRequest();
+    req.setName(name);
+    window.streamService.removeStream(req, new grpc.Metadata(), (err) => {
+      if (err) {
+        console.log('remove stream', { err });
```
Comment (DTCurrie) [inline web/frontend/src/lib/stream.ts:29]:
```suggestion
```

**Review by micheal-parks — APPROVED** [review]
🎉

**Review by mrloureed — APPROVED** [review]
LGTM

**Pull request merged**